dashboard/test/ui/runner.rb

```diff
@@ -337,7 +340,14 @@ def run_tests(env, feature, arguments, log_prefix)
     stderr = stderr.read
     cucumber_succeeded = wait_thr.value.exitstatus == 0
     eyes_succeeded = count_eyes_errors(stdout) == 0
-    return cucumber_succeeded, eyes_succeeded, stdout, stderr, Time.now - start_time
+    duration = Time.now - start_time
+    extra_dimensions = {test_type: test_type,
+                        feature_name: feature}
```

Comment:
Nit: we prefer this layout for hashes in Ruby:
```suggestion
    extra_dimensions = {
      test_type: test_type,
      feature_name: feature
    }
```

Reply:
Yes, it will make the distinction. All that is wrapped on the Infrastructure::Logger.

Right now, there are two dimensions by default (environment and if it is a drone run). 
environemnt -> Dimension needed to avoid the noise you mentioned
is_continuous_integration_run -> Dimension needed to make the distinction between a Drone run vs a test machine not doing Drone runs.

Any other dimensions that are added, are extras and use case specific  (Like here, I am adding the test type)